vuepress/docs/next/tutorials/consume/mt-cds.md

```diff
@@ -0,0 +1,285 @@
+              port: 8081
+              path: /health/health_check
+              scheme: HTTP
+            failureThreshold: 1
+            initialDelaySeconds: 5
+            periodSeconds: 5
+            successThreshold: 1
+            timeoutSeconds: 5
+          env:
+            - name: RUST_LOG
+              value: actix_web=info,actix_server=info,actix_web_middleware_keycloak_auth=trace
+            - name: KEYCLOAK_PUBLIC_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: KC_PUBLIC_KEY
+                  name: YOUR-PUBLIC-KEY
+            - name: CORS_ALLOWED_ORIGIN # use for external CDS service
+              value: All # enter your Entando app domain name
+            - name: CORS_ALLOWED_ORIGIN_END_WITH # use for wildcard domain name
+              value: "$CORS" # enter wildcard domain name for your Entando app, e.g. *domain.com
+          name: cds
+          image: docker.io/entando/cds:1.0.4
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            httpGet:
+              scheme: HTTP
+              port: 8081
+              path: /health/health_check
+            timeoutSeconds: 5
+            successThreshold: 1
+            periodSeconds: 30
+            initialDelaySeconds: 5
+            failureThreshold: 1
+          ports:
+            - containerPort: 8080
+              name: internal-port
+            - containerPort: 8081
+              name: public-port
+          resources:
+            limits:
+              cpu: 500m
+              memory: 500Mi
+            requests:
+              cpu: 500m
+              memory: 500Mi
+          volumeMounts:
+            - mountPath: /entando-data
+              name: cds-data-volume
+      volumes:
+        - name: cds-data-volume
+          persistentVolumeClaim:
+            claimName: YOUR-PVC-CLAIM
+            readOnly: false
+  replicas: 1
+```
+
+### Ingress Descriptor
+``` yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: YOUR-INGRESS-NAME
+  namespace: YOUR-NAMSPACE
+  annotations:
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+      proxy_set_header X-Scheme \$scheme;
+      proxy_set_header X-Forwarded-Proto \$scheme;
+      add_header Content-Security-Policy upgrade-insecure-requests;
+    nginx.ingress.kubernetes.io/proxy-body-size: "150m" # edit according to the file size you require
+
+    nginx.org/client-max-body-size: "150m" # edit according to the file size you require
+spec:
+  $INGRESS_CLASS_FIELD
+  rules:
+    - host: cds-YOUR-APP-NAME.YOUR-HOST-ADDRESS
+      http:
+        paths:
+          - backend:
+              service:
+                name: YOUR-APP-NAME-cds-YOUR-TENANT-CODE-service
+                port:
+                  number: 8081
+            pathType: Prefix
+            path: /$TN
+          - backend:
+              service:
+                name: $APP-cds-$TN-service
```

Comment:
Should we standardize naming conventions (like i see in this attribute) in each yaml or not?

Reply:
@eugeniosant I assume you are talking about strings to be replaced, if yes I guess we should standardize them. At least to make it clear they have to be replaced